build.go

```diff
@@ -360,10 +360,15 @@ func createPackage(options linuxPackageOptions) {
 
 	args := []string{
 		"-s", "dir",
-		"--description", "Grafana",
+		// cloudian customization
+		"--description", "Grafana with some customizations.",
+
 		"-C", packageRoot,
-		"--url", "https://grafana.com",
-		"--maintainer", "contact@grafana.com",
+
+		// cloudian customization
+		"--url", "https://cloudian.com",
+		"--maintainer", "Cloudian Engineering <cloudian-eng@cloudian.com>",
```

Comment:
This is an internal mailinglist email address. We should think what email address we want to use as maintainer email. If we don't want anyone to contact us, we may as well leave it out.

Reply:
@snaik1 any suggestions on what email address would make sense here?

Reply:
Maybe we can create a `packaging@cloudian.com`, but who would read it?

Reply:
Antonette is going to create it.